test/test_all.ipynb

```diff
@@ -0,0 +1,627 @@
+{
```

Comment:
This is super awesome! Thank you so much!

I am wondering if this can be added as unit tests because in that case we can add them to our CI as well, and the checks can be run during PRs. What do you think? (we don't necessarily need to address this in this PR though, this notebook is comprehensive enough for now)

I personally tried when I added PANN, but PANN takes too long to download and it stalls the CI process. VGGish is fast enough so I kept it in the unit test. I am not sure if it takes long to download the CLAP model.

Reply:
Glad you find it useful, I find notebooks much more immediate to understand the code in some cases.

I don't have a specific opinion about adding the code as unit tests. It would be useful but it might also have the same issue with CLAP since each checkpoint is about 2GB.

I think the notebook can work both as a test as well as an easy and quick way for users to copy/paste the section of code they want to use.

If we include a section to the `README` pointing to the notebook, anyone can immediatelly test the code and use what they need.